src/pretix/presale/templates/pretixpresale/event/fragment_cart.html

```diff
@@ -94,6 +94,21 @@
                         </dd>
                     </div>
                 {% endif %}
+                {% if event.settings.show_checkin_number_user and line.checkin_count %}
+                    <div class="cart-icon-details">
+                        <dt class="sr-only">{% trans "Usage:" context "ticket_checkins" %}</dt>
+                        <dd class="text-success">
+                            <span class="fa fa-check-circle fa-fw text-success" aria-hidden="true"></span>
+                            <strong>
```

Comment:
Not sure whether this needs to be `<strong>` as this then gets the same weight as the corresponding item-label. Thanks to coloring it with text-success, maybe a normal font-weight might be enough?

Reply:
I tried that first, but it given that text-success has even less contrast than regular text I felt it was not prominent enough.

My reasoning is this: If you see the order detail page for the first time, this does not appear at all, since the ticket wasn't used, so it does not matter. If you see the order detail page later, this is the thing that **changed** and possible the most important info you are looking for in case of a ticket than can be used X times. So it makes sense to give it a lot of weight.

Reply:
Makes sense!